<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-G0B7K2D" failures="1" tests="8" name="Personal Loan" time="0.053" errors="0" timestamp="2022-05-24T22:36:16 CDT">
  <testcase classname="te.Days1" name="Demo" time="0.008">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at te.Days1.Demo(Days1.java:29)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- Demo -->
  <testcase classname="te.Days2" name="ploan" time="0.001"/>
  <testcase classname="te.Days4" name="LoginAPIhomeloan" time="0.001"/>
  <testcase classname="te.Days4" name="MobileLoginhomeloan" time="0.002"/>
  <testcase classname="te.Days4" name="Webloginhomeloan" time="0.0"/>
  <testcase classname="te.Days4" name="shaheenMOHSIN" time="0.001"/>
  <testcase classname="te.Days4" name="shaheenMOHSIN" time="0.001"/>
  <testcase classname="te.Days4" name="shaheenMOHSIN" time="0.001"/>
  <testcase classname="te.Days1" name="SecondTest" time="0.0">
    <ignored/>
  </testcase> <!-- SecondTest -->
</testsuite> <!-- Personal Loan -->
